config/locales/en.yml

```diff
@@ -684,6 +684,7 @@ en:
       confirmed: "confirmed"
       active_referral: "active"
       add_channel: "Add a web site or youtube channel below to get the promo link!"
+      violation: We regret to inform you that we've suspended your right to participate in Brave Rewards Referral Program due to violation of our <a href="https://basicattentiontoken.org/publisher-terms-of-service/">Terms of Service</a>. However, you can still earn and redeem rewards from from participating in our other programs.
```

Comment:
Looks great Cory! Any chance we could run this copy by catherine corre ?

Reply:
Sent her a message 👍

Reply:
Text changed based on David & Catherine's feedback 
https://github.com/brave-intl/publishers/pull/2002/commits/67ab48f71a096ec69d1d6a1513fe6b33acc752a4